src/Font.elm

```diff
@@ -78,9 +81,30 @@ formatToString format =
         TTF ->
             "truetype"
 
+groupByFamily : List Font -> Dict String (List Font)
```

Comment:
Are you aware of the `Dict.Extra` package? This is quite tricky to read and I suspect `map` and `Dict.Extra.groupBy` would make this job easy peasy.

Reply:
I'd also observe that as written this is triggering my "needs tests" spidey-sense.

Reply:
> "needs tests"
(Although that might change if it were trivial using `Dict.Extra`)

Reply:
Yeah good shout, I've been thinking starting to write unit tests would be good. I've opened #11 to track it.

Reply:
Yeah I'm aware of the "extra" libraries. For now I'm trying to stick to the core libs a) because it's a personal preference, but also b) because I'm hoping people new to Elm can contribute to this repo and I'd like keep the surface area relatively small and simple to start with.

Reply:
👍  No worries, up to you!

For my money the simpler code would be the better experience for the developer, and these `*.Extra` packages are pretty commonly used. If you're reticent about using `Dict.Extra` for now then no worries at all, but I'd recommend reminding yourself to revisit it later (because I am sure you will change your mind :-) ).